### DISCUSSION TITLE
Assembled and found similar clone sequences.

### DISCUSSION THREAD

**Discussion by fookloy** [milaboratory/mixcr#1580] [Q&A] [answered]
HI, i found some sequences that are totally similar but the **Valignscore** and **Calignscore** are different in the assemble output. My sequence should cover the FR1 to FR4.

Why each sequences are same to each other but one still couldn't well align to the C region?

It seems `OseparateByC=true` might be the key parameter account for that, could you please tell how to adjust it? 

<html xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns="http://www.w3.org/TR/REC-html40">
<head>

<meta name=Generator content="Microsoft Excel">

<style>

</style>
</head>
<body>

cloneId | readCount | readFraction | targetSequences | targetQualities | allVHitsWithScore | allDHitsWithScore | allJHitsWithScore | allCHitsWithScore | allVAlignments | allDAlignments | allJAlignments | allCAlignments | nSeqFR1 | minQualFR1 | nSeqCDR1 | minQualCDR1 | nSeqFR2 | minQualFR2 | nSeqCDR2 | minQualCDR2 | nSeqFR3 | minQualFR3 | nSeqCDR3 | minQualCDR3 | nSeqFR4 | minQualFR4 | aaSeqFR1 | aaSeqCDR1 | aaSeqFR2 | aaSeqCDR2 | aaSeqFR3 | aaSeqCDR3 | aaSeqFR4 | refPoints
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
140 | 141 | 1.82E-04 | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAGAGGGTATAGCAGTGGCTGGTTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2804.9) | IGHD6-19*00(85) | IGHJ4*00(430) |   | 80\|384\|405\|0\|304\|SA85GSC95GSA100C\|2953.0 | 22\|39\|63\|304\|321\|\|85.0 | 25\|68\|68\|321\|364\|\|430.0 |   | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAGAGGGTATAGCAGTGGCTGGTTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQESGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CARGYSSGWFDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-1:304:304:-1:-3:321:321:-5:333:364::
1308 | 26 | 3.36E-05 | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAGAGGGTATAGCAGTGGCTGGTTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2804.2) | IGHD6-19*00(85) | IGHJ4*00(430) | IGHM*00(40.1) | 80\|384\|405\|0\|304\|SA85GSC95GSA100C\|2953.0 | 22\|39\|63\|304\|321\|\|85.0 | 25\|68\|68\|321\|364\|\|430.0 |   | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAGAGGGTATAGCAGTGGCTGGTTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQESGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CARGYSSGWFDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-1:304:304:-1:-3:321:321:-5:333:364::
  |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
1092 | 30 | 3.87E-05 | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2773.6) | IGHD6-19*00(81) | IGHJ4*00(420) | IGHM*00(40.9) | 80\|381\|405\|0\|301\|SA85GSC95GSA100C\|2923.0 | 21\|40\|63\|302\|321\|ST24G\|81.0 | 26\|68\|68\|322\|364\|\|420.0 |   | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQESGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CAKGIAVAGLDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-4:301:302:0:-2:321:322:-6:333:364::
88 | 210 | 2.71E-04 | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2774.7) | IGHD6-19*00(81) | IGHJ4*00(420) |   | 80\|381\|405\|0\|301\|SA85GSC95GSA100C\|2923.0 | 21\|40\|63\|302\|321\|ST24G\|81.0 | 26\|68\|68\|322\|364\|\|420.0 |   | CAGGTGCAGCTGCAGGAGTCCGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQESGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CAKGIAVAGLDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-4:301:302:0:-2:321:322:-6:333:364::
  |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
1069 | 31 | 4.00E-05 | CAGGTACAGCTGCAGCAGTCAGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2862) | IGHD6-19*00(81) | IGHJ4*00(420) | IGHM*00(40.5) | 80\|381\|405\|0\|301\|\|3010.0 | 21\|40\|63\|302\|321\|ST24G\|81.0 | 26\|68\|68\|322\|364\|\|420.0 |   | CAGGTACAGCTGCAGCAGTCAGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQQSGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CAKGIAVAGLDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-4:301:302:0:-2:321:322:-6:333:364::
67 | 247 | 3.19E-04 | CAGGTACAGCTGCAGCAGTCAGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCCGGGGACAGTGTCTCTAGCAACAGTGCTGCTTGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGGACATACTACAGGTCCAAGTGGTATAATGATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTACTGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGGGGCCAGGGAACCCTGGTCACCGTCTCCTCAG | [[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[ | IGHV6-1*00(2861.9) | IGHD6-19*00(81) | IGHJ4*00(420) |   | 80\|381\|405\|0\|301\|\|3010.0 | 21\|40\|63\|302\|321\|ST24G\|81.0 | 26\|68\|68\|322\|364\|\|420.0 |   | CAGGTACAGCTGCAGCAGTCAGGTCCAGGACTGGTGAAGCCCTCGCAGACCCTCTCACTCACCTGTGCCATCTCC | 58 | GGGGACAGTGTCTCTAGCAACAGTGCTGCT | 58 | TGGAACTGGATCAGGCAGTCCCCATCGAGAGGCCTTGAGTGGCTGGGAAGG | 58 | ACATACTACAGGTCCAAGTGGTATAAT | 58 | GATTATGCAGTATCTGTGAAAAGTCGAATAACCATCAACCCAGACACATCCAAGAACCAGTTCTCCCTGCAGCTGAACTCTGTGACTCCCGAGGACACGGCTGTGTATTAC | 58 | TGTGCAAAGGGGATAGCAGTGGCTGGTCTTGACTACTGG | 58 | GGCCAGGGAACCCTGGTCACCGTCTCCTCAG | 58 | QVQLQQSGPGLVKPSQTLSLTCAIS | GDSVSSNSAA | WNWIRQSPSRGLEWLGR | TYYRSKWYN | DYAVSVKSRITINPDTSKNQFSLQLNSVTPEDTAVYY | CAKGIAVAGLDYW | GQGTLVTVSS_ | ::::0:75:105:156:183:294:-4:301:302:0:-2:321:322:-6:333:364::

</body>

</html>

`nohup java -Xmx100g -jar /mnt/C_NGS01/Fred/antibody/mixcr.jar align --preset generic-amplicon --species hsa --rna --rigid-left-alignment-boundary --rigid-right-alignment-boundary C -OmergerParameters.minimalOverlap=10 -OmergerParameters.minimalIdentity=0.9 ${id}_1.fq.gz ${id}_2.fq.gz ${id}_align.vdjca`

`nohup java -Xmx100g -jar /mnt/C_NGS01/Fred/antibody/mixcr.jar assemble --assemble-clonotypes-by "{FR1Begin:FR4End}" -OseparateByC=true -ObadQualityThreshold=0 HH-hH_L2_align.vdjca HH-hH_L2_assemble.clns &`

**Comment by mizraelson** [comment, marked as answer]
It's hard to tell without looking at the alignments what went wrong. It could be that only a short C gene sequence is covered, and in some reads, there is a mismatch in that position, which brings the score down, and MiXCR does not identify it with the default thresholds. You can change the parameter to:

`-OseparateByC=false`

This way, MiXCR will not split clones depending on the C gene if the VDJ region is identical.
